src/index.ts

```diff
@@ -1286,6 +1286,15 @@ export class ValidationService {
         return !(this.allowHiddenFields || input.offsetWidth || input.offsetHeight || input.getClientRects().length);
     }
 
+    /**
+     * Checks if the provided input is disabled
+     * @param input
+     * @returns
+     */
+    private isDisabled(input: HTMLElement) {
+        return input.getAttribute('disabled') !== null;
```

Comment:
Using `ValidatableElement` should let you access the various inputs' `disabled` property. I'm pretty sure `getAttribute()` wouldn't correctly account for disabled `fieldset`, but the property might?

```suggestion
    private isDisabled(input: ValidatableElement) {
        return input.disabled;
```